bdata/bdata.py

```diff
@@ -722,7 +722,7 @@ def _correct_deadtime(self, d, deadtime):
             tdwell = self.ppg['dwelltime'].mean * 0.001
              
             # time to do one scan
-            if '2' in self.mode:
+            if '0' in self.mode:
```

Comment:
This will break dwelltime calculations for 2h mode. The line probably should read 

```python
if self.mode in ('20', '2h', '00')
```

Reply:
True - good catch!